experiment/measurer.py

```diff
@@ -61,9 +62,9 @@ def get_experiment_folders_dir():
     return exp_path.path('experiment-folders')
 
 
-def remote_dir_exists(directory: pathlib.Path) -> bool:
-    """Does |directory| exist in the CLOUD_EXPERIMENT_BUCKET."""
-    return filestore_utils.ls(exp_path.gcs(directory), must_exist=False)[0] == 0
+def exists_in_experiment_filestore(path: pathlib.Path) -> bool:
+    """Returns True if |path| exists in the experiment_filestore."""
+    return bool(filestore_utils.ls(exp_path.gcs(path), must_exist=False))
```

Comment:
I may need to change this. It returns `False` for a directory that is empty. That is impossible in GCS, but is possible in a unix filesystem.

Reply:
Done.